src/ratings-and-reviews/components/ReviewBreakdown.test.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import ReviewBreakdown from './ReviewBreakdown.jsx';
+import {starFilters, setStarFilters} from '../index.jsx';
+import {reviewsTotal} from '../index.jsx';
 
 it('Renders bars based on number of ratings', () => {
   const reviews = {1: 25, 2: 10, 3: 50, 4: 70, 5: 100};
-  const app = render(<ReviewBreakdown breakdown={reviews}/>);
+  let starFilters = [];
+  let setStarFiltesr = () => {};
+  const app = render(<ReviewBreakdown breakdown={reviews} starFilters={starFilters} setStarFilters={setStarFilters}/>);
```

Comment:
It looks like you have a typo here! `setStarFiltesr` on line 10 vs. `setStarFilters` on line 11.